pandas/core/tools/timedeltas.py

```diff
@@ -133,31 +134,35 @@ def _validate_timedelta_unit(arg):
         return _unit_map[arg]
     except (KeyError, TypeError):
         if arg is None:
-            return 'ns'
+            return None
         raise ValueError("invalid timedelta unit {arg} provided"
                          .format(arg=arg))
 
 
-def _coerce_scalar_to_timedelta_type(r, unit='ns', box=True, errors='raise'):
+def _coerce_scalar_to_timedelta_type(r, unit=None, box=True, errors='raise'):
     """Convert string 'r' to a timedelta object."""
 
     try:
-        result = convert_to_timedelta64(r, unit)
-    except ValueError:
-        if errors == 'raise':
-            raise
-        elif errors == 'ignore':
-            return r
+        result = parse_timedelta_string(r, unit)
+        result = np.timedelta64(result)
+    except Exception:
```

Comment:
can this be more specific than `Exception`?

Reply:
Changed to `ValueError`